dogen/plugins/cct.py

```diff
@@ -125,6 +125,17 @@ def prepare(self, cfg):
         if 'user' not in cfg['cct']:
             cfg['cct']['user'] = 'root'
 
+        self.install_cct_requirements(cfg)
+
+    def install_cct_requirements(self, cfg):
+        """Ensure that CCT's Python module requirements are installed in the image."""
+        if "packages" not in cfg:
+            cfg["packages"] = []
+        for pkg in ['python-setuptools', 'PyYAML']:
+            if pkg not in cfg["packages"]:
```

Comment:
if that's the case, the generated dockerfile has useless yum instructions in it but they don't fail the build. We can't determine what is in the base image from the cct plugin.

To fix that, we could/should enhance template.jinja to check for each package prior to invoking yum.if that's the case, the generated dockerfile has useless yum instructions in it but they don't fail the build. We can't determine what is in the base image from the cct plugin.

To fix that, we could/should enhance template.jinja to check for each package prior to invoking yum.